bee-message/src/prelude.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+pub use crate::{
+    address::{Address, Ed25519Address, ED25519_ADDRESS_LENGTH},
+    constants::IOTA_SUPPLY,
+    input::{Input, TreasuryInput, UtxoInput},
+    milestone::MilestoneIndex,
+    output::{
+        ConsumedOutput, CreatedOutput, Output, OutputId, SignatureLockedDustAllowanceOutput,
+        SignatureLockedSingleOutput, TreasuryOutput, OUTPUT_ID_LENGTH,
+    },
+    parents::{Parents, MESSAGE_PARENTS_RANGE},
+    payload::{
+        indexation::{HashedIndex, IndexationPayload, HASHED_INDEX_LENGTH},
+        milestone::{
+            MilestoneId, MilestonePayload, MilestonePayloadEssence, MILESTONE_MERKLE_PROOF_LENGTH,
+            MILESTONE_PUBLIC_KEY_LENGTH, MILESTONE_SIGNATURE_LENGTH,
+        },
+        receipt::{MigratedFundsEntry, ReceiptPayload, TailTransactionHash, TAIL_TRANSACTION_HASH_LEN},
+        transaction::{
+            Essence, RegularEssence, RegularEssenceBuilder, TransactionId, TransactionPayload,
+            TransactionPayloadBuilder, TRANSACTION_ID_LENGTH,
+        },
+        treasury::TreasuryTransactionPayload,
+        Payload,
+    },
+    signature::{Ed25519Signature, SignatureUnlock},
+    unlock::{ReferenceUnlock, UnlockBlock, UnlockBlocks},
+    Error, Message, MessageBuilder, MessageId, MESSAGE_ID_LENGTH, MESSAGE_LENGTH_MAX, MESSAGE_LENGTH_MIN,
+};
```

Comment:
I don't like preludes in the first place. If it was only me I'd completely remove this file. It was required by the client teams. At this point I can't track what they are using from it (and partners/exchanges) so I wouldn't touch it to avoid breaking changes.